engine/runtime/src/main/java/org/enso/compiler/pass/analyse/PrivateModuleAnalysis.java

```diff
@@ -30,15 +30,11 @@
  * Inserts errors into imports/exports IRs if the above conditions are violated.
  */
 public final class PrivateModuleAnalysis implements IRPass {
-  private static final PrivateModuleAnalysis singleton = new PrivateModuleAnalysis();
+  public static final PrivateModuleAnalysis INSTANCE = new PrivateModuleAnalysis();
   private UUID uuid;
 
   private PrivateModuleAnalysis() {}
 
-  public static PrivateModuleAnalysis getInstance() {
```

Comment:
Isn't [getter better than a field](http://wiki.apidesign.org/wiki/GettersAndSetters)?

Reply:
In most scenarios, I would say so. But here, I wanted to get as closest to `scalac` as possible - which creates `$MODULE$` public static final field for `object`. Just my preference. I could refactor it to getter once more.